.travis.yml

```diff
@@ -1,7 +1,9 @@
 sudo: false
 language: node_js
 node_js:
-   - "4"                    # Maintenance LTS release
+   - '6'
```

Comment:
I think we discussed dropping support for node 6.x .too as it drops from maintenance in April 2019. Thoughts on that?

Reply:
Looking at https://nodesource.com/node-by-numbers seems like Node 6 usage did not really dip so much. I believe we will still have systems using Node6 and may be it is worth supporting for some more time. Thoughts?

Reply:
On one hand, this major bump can be a good opportunity for Restify 8.x not to be burdened by the need to support node 6.x when it's not maintained starting in April 2019.

On the other hand, those systems running node 6.x would need to be supported and we may need to maintain two Restify release lines (7.x and 8.x) if we go down that route.

I'm fine either way.

I think @retrohacker and @hekike were the ones who suggested to drop 6.x as well. Do you mind sharing your thoughts on dropping support for node 6.x in this major bump?

Reply:
On chatting with @hekike I found that restify codebase seems conservative in order to support older Node versions. The additional burden is that we have had PRs using new language specs and we had to turn them down plus this adds restrictions and prevents us from updating a few npm modules. I think we should go ahead and drop Node6 support too.

Reply:
I don't have any objection as long as we accept that we might need to maintain Restify 7.x for some time in the future.